warrior/Framework/ClassUtils/netconf.py

```diff
@@ -544,7 +544,10 @@ def get(self, filter_string=None, filter_type=None):
             if filter_type == "subtree":
                 xml += "<filter type='subtree'>%s</filter>" % filter_string
             elif filter_type == "xpath":
-                xml += "<filter type='xpath' select='%s'/>" % filter_string
+                if "'" in filter_string:
+                    xml += "<filter type='xpath' select=\"%s\"/>" % filter_string
+                else:
+                    xml += "<filter type='xpath' select='%s'/>" % filter_string
```

Comment:
Hi @venkat1077,

Has the condition where ```filter_string``` contains a double quote (``` " ```) been tested in this PR? I believe if that happens line 550 will throw a ```not well-formed``` exception as well. Please let me know.

Thanks.